static/src/javascripts/projects/common/modules/atoms/youtube-player.js

```diff
@@ -37,6 +34,11 @@ type Handlers = {
     onPlayerStateChange: (event: Object) => void,
 };
 
+let consentState = '';
```

Comment:
Do we need to assign an empty string to the `consentState`, we could just do `let consentState;`.